ignite/cmd/network_chain_launch.go

```diff
@@ -19,9 +19,31 @@ const (
 func NewNetworkChainLaunch() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "launch [launch-id]",
-		Short: "Launch a network as a coordinator",
-		Args:  cobra.ExactArgs(1),
-		RunE:  networkChainLaunchHandler,
+		Short: "Trigger the launch of a chain",
+		Long: `The launch command communicates to the world that the chain is ready to be
+launched.
+
+Only the coordinator of the chain can execute the launch command.
+
+  ignite network chain launch 42
+
+After the launch command is executed no changes to the genesis are accepted. For
+example, validators will no longer be able to successfully execute the "ignite
+network chain join" command to apply as a validator.
+
+The launch command sets the date and time after which the chain will start. By
+default, the current time is set. To give validators more time to prepare for
+the launch, set the time with the "--launch-time" flag:
+
+  ignite network chain launch 42 --launch-time 2023-01-01T00:00:00Z
+
+After the launch command is executed, validators can generate the finalized
+genesis and prepare their nodes for the launch. For example, validators can run
+"ignite network chain prepare" to generate the genesis and populate the peer
+list.
```

Comment:
Should we add something about `revert-launch`, because I assume it is the only way for a coordinator to update the launch time right (`revert-launch` then `launch` again with a different launch time).

Not an expert at all but probably changing the launch time is something that can happen frequently, no ?

Reply:
https://github.com/ignite/cli/pull/3096/commits/b00d69aa33212fd0609fcdd0cae84165fa80b7c5